mapie/classification.py

```diff
@@ -149,9 +146,6 @@ class MapieClassifier(BaseEstimator, ClassifierMixin):
     conformity_scores_: ArrayLike of shape (n_samples_train)
```

Comment:
I think that `conformity_scores_`has been removed to

Reply:
Actually, as this won't be a major release, we should have as few breaking changes as possible. So maybe we should keep those two attributes with a warning saying that users should prefer `mapie_cls.conformity_score.quantiles` to "mapie_cls.quantiles"

Reply:
We should instead add `conformity_score_function_` as a new attribute.

Reply:
At the same time, I can leave `quantiles_` accessible as before.

Reply:
Done 👍

Reply:
and to prevent any breaking changes, you can also add `conformity_scores_' as class attributes

Reply:
It's already done, right?

Reply:
yes indeed ! perfect